yarn-project/aztec-cli/src/index.ts

```diff
@@ -1,21 +1,71 @@
 #!/usr/bin/env node
 import { createLogger } from '@aztec/foundation/log';
 import { Command } from 'commander';
+import { mnemonicToAccount, privateKeyToAccount } from 'viem/accounts';
+import { createDebugLogger } from '@aztec/foundation/log';
+import { createAztecChain, deployL1Contracts } from '@aztec/blockchain';
+import { deployL2Contract } from './deploy_l2_contract.js';
+
+const logger = createDebugLogger('aztec:cli');
 
 const program = new Command();
 const log = createLogger('aztec:aztec-cli');
 
+/**
+ * Function to execute the 'deployRollupContracts' command.
+ * @param rpcUrl - The RPC URL of the ethereum node.
+ * @param apiKey - The api key of the ethereum node endpoint (leave as '' if no api key is used).
```

Comment:
will this comment change after santiagos comment where apiKey can be undefined rather than ''

Reply:
Updated